webpack.config.js

```diff
@@ -1,15 +1,44 @@
+const webpack = require('webpack');
+var InjectHtmlPlugin = require('inject-html-webpack-plugin')
+var PROD = (process.env.NODE_ENV == 1);
+let plugins = [];
+
+if (PROD) {
+    plugins.push(
+        new webpack.optimize.UglifyJsPlugin({
+            minimize: true,
+            sourceMap: true,
+            mangle: {
+                except: ["LibraryView"]
+            }
+        })
+    )
+};
+
+plugins.push(
+    new InjectHtmlPlugin({
+        filename: "./index.html",
+        customInject: [{
+            start: "<!-- start:librarie inject -->",
+            end: "<!-- end:librarie inject -->",
+            content: PROD ? "<script src = './dist/librarie.min.js'></script>" : "<script src = './dist/librarie.js'></script>"
+        }]
+    })
+);
+
 module.exports = {
     entry: [
         "./src/LibraryUtilities.ts",
         "./src/LibraryView.tsx"
     ],
     target: "node",
     output: {
-        filename: "bundle.js",
+        filename: PROD ? "librarie.min.js" : "librarie.js",
         path: __dirname + "/dist",
```

Comment:
One thing though, we want to be releasing `librarie.js` with specific versions. Can we have a variable in this file that tells the version? Say, `let version = "0.0.1"` and then make this `path: __dirname + "/dist/" + version`?